sources/HUBViewController.m

```diff
@@ -411,7 +412,7 @@ - (void)scrollToContentOffset:(CGPoint)contentOffset animated:(BOOL)animated
     const CGFloat x = contentOffset.x;
     const CGFloat y = contentOffset.y - self.collectionView.contentInset.top;
     
-    [self.collectionView setContentOffset:CGPointMake(x, y) animated:animated];
+    [self setContentOffset:CGPointMake(x, y) inCollectionView:self.collectionView animated:animated];
```

Comment:
I was thinking about it. I wanted to make it explicit, but maybe it's better to keep functions short 😄